README.md

```diff
@@ -334,15 +334,16 @@ color       | `00adef` | Specify the color of the video controls. Colors may be
 controls    | `true`   | This parameter will hide all elements in the player (play bar, sharing buttons, etc) for a chromeless experience. ⚠️Warning: When using this parameter, the play bar and UI will be hidden. To start playback for your viewers, you'll need to either enable autoplay or use our player SDK to start and control playback. (available to Plus, PRO, or Business members)
 dnt         | `false`  | Block the player from tracking any session data, including cookies.
 height      |          | The exact height of the video. Defaults to the height of the largest available version of the video.
+keyboard    | `true`   | Allows for keyboard clicks to trigger player events. If false, will disable keyboard activation.
```

Comment:
```suggestion
keyboard    | `true`   | Allows for keyboard input to trigger player events. If false, will ignore keyboard input.
```

Small nit, but "input" or "events" might make more sense than "clicks". Might also be worth leaving a note as well that tabbing will still be supported when keyboard input/events are disabled.